src/components/common/Summary/Summary.js

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+
+import styles from './Summary.module.scss';
+
+const AntTabs = withStyles({
+  indicator: {
+    backgroundColor: '#ffffff',
+  },
+})(Tabs);
+
+const AntTab = withStyles(theme => ({
+  root: {
+    borderRight: '1px solid #b6b6b6',
+    borderBottom: '1px solid #b6b6b6',
+    textAlign: 'center',
+    textTransform: 'uppercase',
+    color: 'black',
+    minWidth: 72,
+    fontWeight: 700,
+    paddingRight: theme.spacing(4),
+    paddingLeft: theme.spacing(4),
+    fontFamily: [
+      '-apple-system',
+      'BlinkMacSystemFont',
+      '"Segoe UI"',
+      'Roboto',
+      '"Helvetica Neue"',
+      'Arial',
+      'sans-serif',
+      '"Apple Color Emoji"',
+      '"Segoe UI Emoji"',
+      '"Segoe UI Symbol"',
+    ].join(','),
+    '&$selected': {
+      color: '#d58e32',
+      fontWeight: 700,
+      borderBottom: '1px solid #000000',
+    },
+    '&:focus': {
+      color: '#d58e32',
+      borderBottom: 'none',
+    },
+  },
+  selected: {},
+}))(props => <Tab disableRipple {...props} />);
+
+const StyledTab = withStyles(theme => ({
+  root: {
+    textTransform: 'none',
+    color: '#fff',
+    fontWeight: theme.typography.fontWeightRegular,
+    fontSize: theme.typography.pxToRem(15),
+    marginRight: theme.spacing(1),
+    '&:focus': {
+      opacity: 1,
+    },
+  },
+}))(props => <Tab disableRipple {...props} />);
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+  },
+  padding: {
+    padding: theme.spacing(3),
+  },
+  demo1: {
+    backgroundColor: theme.palette.background.paper,
+  },
+  demo2: {
+    backgroundColor: '#2e1534',
+  },
+}));
+
+export default function Summary() {
+  function TabPanel(props) {
+    const { children, value, index, ...other } = props;
+
+    return (
+      <div
+        role='tabpanel'
+        hidden={value !== index}
+        id={`simple-tabpanel-${index}`}
+        aria-labelledby={`simple-tab-${index}`}
+        {...other}
+      >
+        {value === index && (
+          <Box p={3}>
+            <Typography>{children}</Typography>
+          </Box>
+        )}
+      </div>
+    );
+  }
+
+  TabPanel.propTypes = {
+    children: PropTypes.node,
+    index: PropTypes.any.isRequired,
+    value: PropTypes.any.isRequired,
+  };
+
+  function a11yProps(index) {
+    return {
+      id: `simple-tab-${index}`,
+      'aria-controls': `simple-tabpanel-${index}`,
+    };
+  }
+
+  const useStyles = makeStyles(theme => ({
+    root: {
+      flexGrow: 1,
+      backgroundColor: theme.palette.background.paper,
+    },
+  }));
+
+  const classes = useStyles();
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
```

Comment:
To chyba obsługuje zmianę aktywnej zakładki, to taki kod powinien być zamknięty w komponencie z Tabami a nie Summary. Jeśli w innym komponencie ponownie będziemy chcieli użyć tabów to bez sensu pisać ten sam kod do obsługi kliknięcia. Komponent który używa tabów nie musi wiedzieć w jaki sposób się one przełączają.

Reply:
To znajdue sie w Tabs